pkg/tnf/handlers/shutdown/shutdown.gotemplate

```diff
@@ -0,0 +1,3 @@
+{{- range .spec.containers -}}
```

Comment:
The point of this is just to make generic test return something more readable than obscure patterns?

Reply:
Get it now, it also does a loop. Nice.

Reply:
Yes exactly. otherwise, it will return the command defined in the output or some "<no value>".

Reply:
This is nice